app/template/nav.php

```diff
@@ -57,10 +61,12 @@
                 </li>
             <?php endif ?>
         </ul>
-        <ul>
-            <li>
-                <?php include app_path() . "/components/theme_switcher.php" ?>
-            </li>
-        </ul>
+        <div class="icon-buttons">
+            <?php include app_path() . "/components/theme_switcher.php" ?>
+            <a href="/feedbacks" role=button class="outline contrast" aria-label="auto" aria-live="polite"
+                title="Feedbacks">
```

Comment:
I suggest using the same name as page title here
```suggestion
                title="Bugs et suggestions">
```

Reply:
Done